src/Nri/Ui/ClickableText/V3.elm

```diff
@@ -235,7 +235,8 @@ hideTextFor : MediaQuery -> Attribute msg
 hideTextFor mediaQuery =
     css
         [ Css.Media.withMedia [ mediaQuery ]
-            [ Css.Global.descendants
+            [ Css.borderStyle Css.none |> Css.important
```

Comment:
❓ what is this border style removal for?

Reply:
for removing border when text is hidden https://noredink.slack.com/archives/C02NMHB1K55/p1674230782897309?thread_ts=1674139035.616359&cid=C02NMHB1K55